sdk/core/src/modify_conditions.rs

```diff
@@ -13,7 +13,7 @@ pub enum IfSinceCondition {
 }
 
 impl IfSinceCondition {
-    pub(crate) fn add_header(&self, builder: Builder) -> Builder {
```

Comment:
Why rename these? We can probably just get rid of them entirely since we can use their implementation of `AddAsHeader` instead, no?

Reply:
Yes I've just renamed these in bulk with a brutal `sed` script 🪓. I haven't bothered to correct them since they are obsolete and will be removed as soon as the remaining code no longer references them.